### ISSUE TITLE
escaping of paths in spaces doesn't seem to be required (darwin)

### ISSUE THREAD

**Issue by boneskull** [krampstudio/grunt-jsdoc#119]
In fact, it causes an error.

```
>> An error occurs in jsdoc process:
>> fs.js:502
>>   return binding.open(pathModule._makeLong(path), stringToFlags(flags), mode);
>>                  ^
>> Error: ENOENT, no such file or directory '/Volumes/alien/projects/boneskull/intro-to-js-unit-testing/working-files/Chapter\ 2/0207\ Overview\ of\ Test-Driven\ Development/README.md'
>>     at Error (native)
>>     at Object.fs.openSync (fs.js:502:18)
>>     at Object.fs.readFileSync (fs.js:354:15)
>>     at new ReadMe (/Volumes/alien/projects/boneskull/intro-to-js-unit-testing/working-files/node_modules/grunt-jsdoc/node_modules/jsdoc/lib/jsdoc/readme.js:20:22)
>>     at buildSourceList (/Volumes/alien/projects/boneskull/intro-to-js-unit-testing/working-files/node_modules/grunt-jsdoc/node_modules/jsdoc/cli.js:291:26)
>>     at Object.module.exports.cli.scanFiles (/Volumes/alien/projects/boneskull/intro-to-js-unit-testing/working-files/node_modules/grunt-jsdoc/node_modules/jsdoc/cli.js:308:18)
>>     at module.exports.cli.main (/Volumes/alien/projects/boneskull/intro-to-js-unit-testing/working-files/node_modules/grunt-jsdoc/node_modules/jsdoc/cli.js:231:9)
>>     at Object.module.exports.cli.runCommand (/Volumes/alien/projects/boneskull/intro-to-js-unit-testing/working-files/node_modules/grunt-jsdoc/node_modules/jsdoc/cli.js:192:5)
>>     at /Volumes/alien/projects/boneskull/intro-to-js-unit-testing/working-files/node_modules/grunt-jsdoc/node_modules/jsdoc/jsdoc.js:182:13
>>     at Object.<anonymous> (/Volumes/alien/projects/boneskull/intro-to-js-unit-testing/working-files/node_modules/grunt-jsdoc/node_modules/jsdoc/jsdoc.js:184:3)
Warning: jsdoc failure Use --force to continue.
```

Commenting out `tasks/jsdoc-plugin:52-54` solves the issue.

It was also incorrect in the first place--these lines only escape the *first* space found--if there are multiple in the path, the rest would not have been escaped.

I'll see if I can send a PR.

**Comment by krampstudio** [comment]
Oh, yes, I'll try to find the previous PR that had introduced this issue (I guess it was for windows).

**Comment by krampstudio** [comment]
This should be fixed in 0.6.5.

**Comment by krampstudio** [comment]
Please reopen if not fixed.
